analysis/topeft_run2/analysis_processor.py

```diff
@@ -487,7 +487,8 @@ def process(self, events):
             ######### Masks we need for the selection ##########
 
             # Get mask for events that have two sf os leps close to z peak
-            sfosz_3l_mask = tc_es.get_Z_peak_mask(l_fo_conept_sorted_padded[:,0:3],pt_window=10.0)
+            sfosz_3l_OnZ_mask = tc_es.get_Z_peak_mask(l_fo_conept_sorted_padded[:,0:3],pt_window=10.0)
+            sfosz_3l_OffZ_low_mask = tc_es.get_off_Z_mask_low(l_fo_conept_sorted_padded[:,0:3],pt_window=0.0)
```

Comment:
I left some comments on this function in https://github.com/TopEFT/topcoffea/pull/10. We could certainly leave this as is with no issues, or modify it to be more general, or even move it into _this_ repo if we decide it is analysis specific enough.